source/Halibut.Tests/Support/Logging/InMemoryLog.cs

```diff
@@ -2,11 +2,11 @@
 using System.Collections.Concurrent;
 using System.Collections.Generic;
 using Halibut.Diagnostics;
-using Serilog;
+using Halibut.Diagnostics.LogWriters;
 
 namespace Halibut.Tests.Support.Logging
 {
-    public class InMemoryLog : ILog
+    public class InMemoryLog : ILog, ILogWriter
```

Comment:
Nit: I think making a separate `InMemoryLogWriter` would make the tests a little clearer to understand. Otherwise, people might look at the tests you are using this class as an `ILogWriter` for, and wonder "why we are using an `ILog`?"

Reply:
Maybe the same goes for `TestContextConnectionLog` as well?

Reply:
done for InMemoryLog only